rds/README.md

```diff
@@ -117,3 +117,121 @@ function generate_rds_password () {
+  EXECUTE format('GRANT CONNECT ON DATABASE %s TO %s_%s_fivetran;', database_name, project, environment);
+  EXECUTE format('GRANT USAGE ON SCHEMA public TO %s_%s_fivetran;', project, environment);
+
+  -- add permissions for tables listed above
+  FOREACH table_name IN ARRAY allowed_tables
+  LOOP
+    EXECUTE format('GRANT SELECT ON TABLE %s TO %s_%s_fivetran;', table_name, project, environment);
+  END LOOP;
+
+  -- template for adding per column permissions, please note the XMIN column at the end (needed when you use XMIN)
+  EXECUTE format('GRANT SELECT (id, ..., xmin) on <table_name> to %s_%s_fivetran;', project, environment);
+END
+$body$
+```
+
+#### XMIN
+
+XMIN is not suitable for large databases. It does not detect deletes.
+
+There is no additional setup required.
+
+#### Logical replication using WAL / test_decoding plugin
+
+We need to create a replication slot to enable usage of `test_decoding` plugin:
+
+1. Set enable_replication = true in RDS terraform
+```terraform
+module "rds" {
+  source = "github.com/dbl-works/terraform//rds?ref=x"
+  ...
+  enable_replication = true
+}
+```
+2. Apply with: `tf apply`
+3. Manually restart the DB via AWS console to apply static parameters. Please note that DB will be offline for some time (should be minutes), so this is best executed during low traffic.
+4. You can check state of replication with:
+```SQL
+SHOW rds.logical_replication;
+```
+5. Create replication slot by executing following SQL as user with supervisor privileges:
+```SQL
+SELECT pg_create_logical_replication_slot('fivetran_replication_slot', 'test_decoding');
+GRANT rds_replication TO <fivetran_db_username>;
+```
+6. Check if slot is available (it may take some time for entries to appear)
+```SQL
+SELECT count(*) FROM pg_logical_slot_peek_changes('fivetran_replication_slot', null, null);
+```
+
+NOTE: After you create the slot it has to be used by Fivetran. Otherwise the WAL data will accumulate and take all available DB space. Once DB is full, it is not usable!
+
+#### AWS security policy
+
+If the database in publicly accessible, it may be protected by inbound rules. To allow Fivetran access to the DB, please whitelist [Fivetran IPs](https://fivetran.com/docs/getting-started/ips). Our Fivetran is [hosted in zone](https://fivetran.com/dashboard/warehouse) `europe-west3 (Frankfurt)`, so we need to whitelist:
+```
+35.235.32.144/29
+```
+#### SSH tunneling
+
+If the application DB is behind bastion, an SSH tunnel has to be configured to allow access:
+
+1. Bastion URL is the same you use to access the database.
+2. Please read how to setup bastion with public key [here](https://github.com/dbl-works/terraform/pull/118/files#diff-94cb490f08aaef66dedfd461d69ed21f92555c728a81212e41accf096b31c118)
+
+Check [Modifly connector](https://fivetran.com/integrations/postgres_rds/?name=modifly&groupId=plausible_comprise) for example.
```

Comment:
let's write some generic example here

Reply:
I don;t have experience, can you add an example @swiknaba?